library/date/iso8601_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../spec_helper'
+require 'date'
+
+describe "Date#iso8601" do
+  class StringSubclass < String; end
```

Comment:
Could you use `Class.new(String)` instead, where it is used below?
Declaring it like this means it's defining `StringSubclass` at the top-level.